code/_onclick/observer.dm

```diff
@@ -61,6 +61,8 @@
 
 // health + cyborg analyzer for ghosts
 /mob/living/attack_ghost(mob/dead/observer/user)
+	if(!istype(user))
```

Comment:
Let's make it a bit more obvious to people reading this in the future why the type check is needed.
```suggestion
	// Revenants use attack_ghost when they click on things, but they are not observers and don't have .health_scan .
	if(!istype(user))
```

Reply:
Sure why not.